sklearn/base.py

```diff
@@ -369,7 +369,7 @@ class RegressorMixin:
     _estimator_type = "regressor"
 
     def score(self, X, y, sample_weight=None):
-        """Returns the coefficient of determination R^2 of the prediction.
+        """Return the coefficient of determination R^2 of the prediction.
```

Comment:
I think the standard is to have `Returns` in plural

Reply:
This was causing numpydoc to fail with the following error:

`SS05: Summary must start with infinitive verb, not third person (e.g. use "Generate" instead of "Generates")`

I also thought that it should be Returns, but made this change to get all tests to pass.

Reply:
Huh, `interesting`.

It looks like sklearn convention is slightly at odds with numpydoc convention. In that case, sure, do the thing that makes the tests pass. Maybe this convention will be re-examined in the future